### PR TITLE
Update wrapper for Nix

### PR DESCRIPTION
**Pull request by lionello**
Also moving the `export` statements to the top of the file for `gpg-agent`.

### CODE REVIEW

File: internal/jsonapi/manifest/wrapper.go [hunk 1]
```diff
@@ -12,17 +12,18 @@ import (
 
 const wrapperTemplate = `#!/bin/sh
 
+export PATH="$PATH:$HOME/.nix-profile/bin" # required for Nix
```
Comment (dominikschulz) [inline internal/jsonapi/manifest/wrapper.go:15]: I think we should wrap it into a `if [ -d <DIR> ];` statement to only update the PATH if the destination does exist. wdyt?

**Review by dominikschulz — CHANGES_REQUESTED** [review]

File: internal/jsonapi/manifest/wrapper.go (hunk 1 above)
Comment (lionello) [reply internal/jsonapi/manifest/wrapper.go:15]: Hmm, that makes sense, but would duplicate all the paths, unless we put them in variables, etc.. Having non-existing paths in `$PATH` seems to have no negative effects, so I kinda like the shorter code like this.

Comment (dominikschulz) [reply internal/jsonapi/manifest/wrapper.go:15]: I don't mind duplicating them. But you can also put them into variables if you prefer.

Comment (lionello) [reply internal/jsonapi/manifest/wrapper.go:15]: I've checked and there's no problem with having non-existent paths in `PATH` so I think the simpler code is the better code. We should leave it as-is (and only add the Nix path and move them up so `gpg-agent` is found.)

**Comment by lionello** [comment]
I can't see why `CommitCheck` is failing. When I press `Details` it just throws me back to this PR page.

**Review by dominikschulz — APPROVED** [review]
LGTM

**Pull request merged**